src/Expression/Builder/ExpressionBuilderInterface.php

```diff
@@ -16,20 +18,20 @@
  *
  * @see ExpressionInterface
  *
- * @template T as ExpressionInterface
+ * @template T as T
```

Comment:
```suggestion
 * @template ExpressionInterface as ExpressionInterface
```

Looks better to use `ExpressionInterface`?

Reply:
Oops. This is strange artifact of PhpStorm refactoring =)
Fixed.